### PR TITLE
docs: Pull request & issues templates

### PR DESCRIPTION
**Pull request by jacopobacci**
I created a PR template and 2 issues template, one for new tasks and one for bugs. Feel free to suggest changes.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+1. [Step 1]
+2. [Step 2]
+3. [Step 3]
+   ...
+
+### Environment
+
+- Operating System: [e.g., Windows 10, macOS, Linux]
+- Python Version: [e.g., Python 3.8.5]
```
Comment (BonnetAdam) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]: Python version should only be 3.10

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+## Related Issues
+
+[If this pull request resolves any open issues, mention them here.]
+
+## Checklist:
+
+- [ ] **Code Review**: Have you reviewed your code to ensure it follows best practices and meets the project's coding standards?
+- [ ] **Testing**: Have you tested your changes locally to ensure they work as expected?
+- [ ] **Documentation**: Have you updated any relevant documentation or comments to reflect your changes?
+- [ ] **Pre-commit Checks**: Have you ensured that pre-commit checks have automatically run upon committing to catch any potential issues?
```
Comment (BonnetAdam) [inline .github/pull_request_template.md:14]: These one might not be necessary... If GitHub action is not successful they cannot create PR

**Review by BonnetAdam — COMMENTED** [review]
Other than that all good for me

File: .github/pull_request_template.md (hunk 2 above)
Comment (jacopobacci) [reply .github/pull_request_template.md:14]: it's a self proofreading checklist. it's just fo you to over check your work before creating the pr. if you want i rmeove it no worries :)

Comment (BonnetAdam) [reply .github/pull_request_template.md:14]: No it's find to me

**Review by BonnetAdam — APPROVED** [review]

**Pull request merged**